File: UNIDAD 1/EXAMEN_EU3_GR10/VIAJECITOS_REST_DOTNET_GR10/02. CLIESC/VIAJECITOS_REST_CLIESC_GR10/VIAJECITOS_REST_CLIESC_GR10/Modelos/Vuelo.cs
using System;

namespace VIAJECITOS_REST_CLIESC_GR10.Modelos
{
    public class Vuelo
    {
        public int IdVuelo { get; set; }
        public string CiudadOrigen { get; set; }
        public string CiudadDestino { get; set; }
        public decimal Valor { get; set; }
        public DateTime HoraSalida { get; set; }
    }
}

File: UNIDAD 1/00_SOA_SIN_BDD/00. SOAP_JAVA_CONUNI_GR10/02. CLIMOV/app/src/main/java/ec/edu/monster/modelo/Vuelo.kt
package ec.edu.monster.modelo

import java.io.Serializable
import java.math.BigDecimal

data class Vuelo(
    val idVuelo: Int,
    val ciudadOrigen: String,
    val ciudadDestino: String,
    val valor: BigDecimal,
    val horaSalida: String
) : Serializable